ds/scripts/install/redis-config.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -x
+source /host/settings.sh
+
+if [ -n "$REDIS_HOST" ];
+then
```

Comment:
Make the test like this: `[[ -n $REDIS_HOST ]] || exit 0` . This makes the code more readable.

Reply:
And also remove `if..then`.